src/mame/act/victor9k.cpp

```diff
@@ -572,9 +574,13 @@ WRITE_LINE_MEMBER( victor9k_state::via2_irq_w )
 void victor9k_state::via3_pb_w(uint8_t data)
 {
 	// codec clock output
-	m_ssda->rx_clk_w(!BIT(data, 7));
-	m_ssda->tx_clk_w(!BIT(data, 7));
+	m_ssda->rx_clock_w(!BIT(data, 7));
+	m_ssda->tx_clock_w(!BIT(data, 7));
 	m_cvsd->clock_w(!BIT(data, 7));
+
+	long now = machine().time().as_double();
+	LOGSOUND("VICTOR9K tx_clk_w %02x %d\n", !BIT(data, 7),  now);
```

Comment:
Isn’t this going to truncate the fractional part?  Logging with seconds granularity doesn’t seem particularly useful here.

Reply:
I need to clean up the logging altogether. This was me debugging.